src/libs/SearchUtils.ts

```diff
@@ -833,6 +840,63 @@ function getOverflowMenu(itemName: string, hash: number, inputQuery: string, sho
     ];
 }
 
+function getIDFromDisplayValue(filterName: string, filter: string | string[], cardList: OnyxTypes.CardList, reports: OnyxCollection<OnyxTypes.Report>, taxRates: Record<string, string[]>) {
```

Comment:
I don't have very strong feelings, but perhaps:
```suggestion
function findIDByDisplayValue(filterName: string, filter: string | string[], cardList: OnyxTypes.CardList, reports: OnyxCollection<OnyxTypes.Report>, taxRates: Record<string, string[]>) {
```

cos we're not "getting" it from the value - we use the value to find is somewhere.